tests/integration/test_target_snowflake.py

```diff
@@ -246,6 +246,25 @@ def assert_logical_streams_are_in_snowflake_and_are_empty(self):
         self.assertEqual(table_three, [])
         self.assertEqual(table_four, [])
 
+    def assert_binary_data_are_in_snowflake(self, should_metadata_columns_exist=False):
+        # Get loaded rows from tables
+        snowflake = DbSync(self.config)
+        target_schema = self.config.get('default_target_schema', '')
+        table_one = snowflake.query("SELECT * FROM {}.test_binary ORDER BY ID".format(target_schema))
+
+        # ----------------------------------------------------------------------
+        # Check rows in table_one
+        # ----------------------------------------------------------------------
+        expected_table_one = [
+            {'ID': b'pk2', 'DATA': b'data2', 'CREATED_AT': datetime.datetime(2019, 12, 17, 16, 2, 55)},
+            {'ID': b'pk4', 'DATA': b'data4', "CREATED_AT": datetime.datetime(2019, 12, 17, 16, 32, 22)},
```

Comment:
I think that's a good idea to stick to one type of quotes. We don't have pylint here yet, right?

Reply:
Yes, we do have Pylint, afaik, it doesn't care about which type of quotes is used.